### PR TITLE
fix(tier4_perception_launch): update multi-channel subscribing channel name to lidar_detection_model_type

### PR DESCRIPTION
**Pull request by technolojin**
## Description

The object detection channels in the `tracking.launch.xml` file have been updated to use the lidar detection model type.

Implementing changes by PR https://github.com/autowarefoundation/autoware.universe/pull/9003

This PR do not contain logic change.

## Related links

**Parent Issue:**

- Link

## How was this PR tested?

set parameters as following
* `use_multi_channel_tracker_merger` true
* `lidar_detection_model` centerpoint/centerpoint_tiny

## Notes for reviewers

None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

**Review by yukkysaito — APPROVED** [review]
@YoshiRi The change itself looks good, but I haven’t been able to check the coding style or detailed logic. Could I ask you to review those?

File: launch/tier4_perception_launch/launch/object_recognition/tracking/tracking.launch.xml [hunk 1]
```diff
@@ -69,20 +69,24 @@
   <group if="$(var use_multi_channel_tracker_merger)">
     <let name="detection_enabled_channels" value="$(var input/detection_input_channels)"/>
     <group scoped="false" if="$(eval '&quot;$(var mode)&quot;==&quot;camera_lidar_radar_fusion&quot;')">
-      <let name="detection_enabled_channels" value="['lidar_$(var lidar_detection_model)_validated','camera_lidar_fusion','detection_by_tracker','radar_far']" if="$(var use_detection_by_tracker)"/>
-      <let name="detection_enabled_channels" value="['lidar_$(var lidar_detection_model)_validated','camera_lidar_fusion','radar_far']" unless="$(var use_detection_by_tracker)"/>
+      <let
```
[the remark this answers is not in the document]
Comment (technolojin) [reply launch/tier4_perception_launch/launch/object_recognition/tracking/tracking.launch.xml:72]:
```
prettier launch.xml......................................................Failed
- hook id: prettier-launch-xml
- files were modified by this hook
launch/tier4_perception_launch/launch/object_recognition/tracking/tracking.launch.xml
```

File: launch/tier4_perception_launch/launch/object_recognition/tracking/tracking.launch.xml [hunk 2]
```diff
@@ -69,20 +69,24 @@
...
+        name="detection_enabled_channels"
+        value="['lidar_$(var lidar_detection_model_type)_validated','camera_lidar_fusion','detection_by_tracker','radar_far']"
+        if="$(var use_detection_by_tracker)"
+      />
+      <let name="detection_enabled_channels" value="['lidar_$(var lidar_detection_model_type)_validated','camera_lidar_fusion','radar_far']" unless="$(var use_detection_by_tracker)"/>
```
Comment (YoshiRi) [inline launch/tier4_perception_launch/launch/object_recognition/tracking/tracking.launch.xml:77]: LiDAR detection validation is optional. See `use_validator` parameter used in detection.launch.xml.
So, I think we need to add `'lidar_$(var lidar_detection_model_type)'` in every part of this launch.

**Review by YoshiRi — COMMENTED** [review]
Here is trivial comment. Since we normally use validation it won't effect most projects.

File: launch/tier4_perception_launch/launch/object_recognition/tracking/tracking.launch.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (YoshiRi) [reply launch/tier4_perception_launch/launch/object_recognition/tracking/tracking.launch.xml:72]: wow... Then you can ignore my comment.

File: launch/tier4_perception_launch/launch/object_recognition/tracking/tracking.launch.xml (hunk 2 above)
Comment (technolojin) [reply launch/tier4_perception_launch/launch/object_recognition/tracking/tracking.launch.xml]: thank you for the comment. it is fixed https://github.com/autowarefoundation/autoware.universe/pull/9624/commits/7dc931bc50d8a6cda3ea73b7346f08fb3a8afdbf

**Review by YoshiRi — APPROVED** [review]
LGTM

**Pull request merged**